starter-code/bin/seeds.js

```diff
@@ -1 +1,26 @@
 // Iteration #1
+const mongoose = require("mongoose");
+const Drone = require("../models/drones");
+//mongoose.connect("mongodb://localhost/drones-dev");
+
+const droneData = [
+  { name: "Racer 57", propellers: 4, maxSpeed: 20 },
+  { name: "Courier 3000i", propellers: 6, maxSpeed: 18 }
+];
+
+droneData.forEach(droneObject => {
```

Comment:
I was wrong about this one. Apparently you CAN pass an array to `Model.create`. You can try it out if you want.
Sorry about that :)